### PR TITLE
Add unit tests for cache_operator

### PR DESCRIPTION
**Pull request by mark-dong2023**
### Description
Add unit tests for cache_operator
### Test Plan

unit tests

### CODE REVIEW

**Review by larry-aptos — APPROVED** [review]
nice! LGTM

File: ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs [hunk 1]
```diff
@@ -507,4 +507,71 @@ mod tests {
...
+    async fn cache_update_cache_transactions_ok() {
+        let mut transactions = vec![];
+        let version = 123_u64;
+        let encoded_proto_data = String::from("123");
+        let timestamp_in_seconds = 12_u64;
+        transactions.push((version, encoded_proto_data, timestamp_in_seconds));
+        let cmds = vec![MockCmd::new(
+            redis::cmd("SET")
+                .arg(version)
+                .arg(String::from("123"))
```
Comment (rtso) [inline ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs:537]: nit: use `encoded_proto_data` here

File: ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs [hunk 2]
```diff
@@ -507,4 +507,71 @@ mod tests {
...
+        let mut transactions = vec![];
+        let version = CACHE_SIZE_EVICTION_LOWER_BOUND + 100;
+        let encoded_proto_data = String::from("123");
+        let timestamp_in_seconds = 12_u64;
+        transactions.push((version, encoded_proto_data, timestamp_in_seconds));
+        let mut redis_pipeline = redis::pipe();
+        redis_pipeline
+            .cmd("SET")
+            .arg(version)
+            .arg(String::from("123"))
```
Comment (rtso) [inline ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs:562]: nit: use `encoded_proto_data` here

**Review by rtso — APPROVED** [review]

File: ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs (hunk 2 above)
Comment (mark-dong2023) [reply ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs:562]: good catch

**Pull request merged**